diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -112,8 +112,8 @@ data class CommonConfiguration(private val configuration: Map<String, String>?)
     /**
      * Start of package name, which shoould be common, e.g. org.example.myproject
      */
-    val domainName: String by lazy {
-        (configuration ?: emptyMap()).getOrDefault("domainName", "")
+    val domainName: String? by lazy {
+        (configuration ?: emptyMap())["domainName"]
```

Comment:
```suggestion
        configuration?.et("domainName")
```
Simply this will return `null` when coniguration is missing.